site/en/guide/distributed_training.ipynb

```diff
@@ -636,7 +636,7 @@
       "source": [
         "## Use tf.distribute.Strategy with custom training loops\n",
         "\n",
-        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](/keras/writing_a_training_loop_from_scratch.ipynb).\n",
+        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](./keras/writing_a_training_loop_from_scratch.ipynb).\n",
```

Comment:
Keras guides under `/guide` should have full links, they are hosted on keras.io. Fixed this:

```suggestion
        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](https://www.tensorflow.org/guide/keras/writing_a_training_loop_from_scratch).\n",
```